.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,71 @@
+# - When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# - When a job is added/removed/renamed, please make corresponding changes in ci-docs.yaml.
```

Comment:
Will this be a required workflow as part of branch protections? If so, we need to create a docs version if we're ignoring the docs and contrib folder.

Reply:
I think for now workflow still needs to run the full CI.

Reply:
Ok, so no branch protection requirement to run this workflow, right (aka PRs don't need this workflow to pass in order to be merged)?

Reply:
Got it. That sounds good! Thanks for confirming